wear/src/main/res/values/strings.xml

```diff
@@ -27,6 +27,15 @@
         <item>1</item>
         <item>2</item>
     </string-array>
+    
+    <string-array name="dex_slot">
+        <item>1</item>
+        <item>2</item>
+    </string-array>
+    <string-array name="dex_slot_values">
```

Comment:
Why is the second array necessary. It is identical to the first one.

Please can you also add `3` for customized transmitters?

Reply:
What’s a customized transmitter? Haven’t heard of it? Alternative Firmware? More slots ?

Reply:
> What’s a customized transmitter? Haven’t heard of it? Alternative Firmware? More slots ?

PM me

Reply:
Will add slot3 asap